js/os.ts

```diff
@@ -19,8 +20,7 @@ export function codeFetch(
   moduleSpecifier: string,
   containingFile: string
 ): ModuleInfo {
-  console.log("Hello from codeFetch");
-
+  util.log("os.ts codeFetch", moduleSpecifier, containingFile);
```

Comment:
small nit:
maybe we can just call `log` here?
by adding `import { assert, log } from "./util"` like you did in `js/main.ts`